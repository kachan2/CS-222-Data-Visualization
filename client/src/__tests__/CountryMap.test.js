import React from "react";
import ShallowRenderer from 'react-test-renderer/shallow';

// Components
import MapChart from "../maps/CountryMap"

it('renders correctly react-test-renderer', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<MapChart />);
  const result = renderer.getRenderOutput();

  expect(result).toMatchSnapshot();
});