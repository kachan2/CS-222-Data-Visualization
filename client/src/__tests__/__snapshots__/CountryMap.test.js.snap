// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly react-test-renderer 1`] = `
<ComposableMap
  projection="geoAlbersUsa"
>
  <Geographies
    geography="https://cdn.jsdelivr.net/npm/us-atlas@3/counties-10m.json"
  >
    [Function]
  </Geographies>
</ComposableMap>
`;
